reactor-netty-http/src/test/java/reactor/netty/ByteBufFluxTest.java

```diff
@@ -65,10 +60,10 @@ private void doTestByteBufFluxFromPath(boolean withSecurity) throws Exception {
 		}
 
 		Path path = Paths.get(getClass().getResource("/largeFile.txt").toURI());
-		DisposableServer c = server.handle((req, res) ->
-		                                      res.send(ByteBufFlux.fromPath(path))
-		                                         .then())
-		                           .bindNow();
+		disposableServer= server.handle((req, res) ->
```

Comment:
```suggestion
		disposableServer = server.handle((req, res) ->
```

Reply:
This was fixed with a previous commit so I just rebased.